chore(docs): add troubleshooting note when installation says Cython is not available (#4790)
DataDog/dd-trace-py: docs/troubleshooting.rst

Troubleshooting
===============

Installation failure
====================

``ModuleNotFoundError: No module named 'Cython'``

> pip is failing to install `dd-trace` and complaining about a missing module (Cython).

Reviewer: ```suggestion
pip is failing to install ``ddtrace`` and complaining about a missing module (Cython).
```

Author: My bad, thanks for reviewing this and submitting improvements.

> Ensure that pip is installed with version 18 or above. 

Reviewer: ```suggestion
``pip>=18`` is required to properly install ``ddtrace`` package.

Check which version of pip you are using with ``pip --version``.

Consider upgrading pip via ``pip install -U pip>=18``.
```

Troubleshooting
===============

Installation failure
====================

``ModuleNotFoundError: No module named 'Cython'``

> pip is failing to install `ddtrace` and complaining about a missing module (Cython).

Reviewer: ```suggestion
pip is failing to install ``ddtrace`` and complaining about a missing module (Cython).
```

this is rST not markdown, so we need double back ticks
